plugins/catalog-backend-module-system-entity-model/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-system-entity-model",
```

Comment:
Curious to know why call it 'system-entity-model' instead of 'default-entity-model' or 'entity-model' only. My understanding of Catalog's "system" meaning may be unclear.

Reply:
It's been referred to "the system entity model" and SEM for short since the beginnings internally, but it could be argued that that was more clear internally than externally when open sourcing it?

I'm open to discussing renaming it. Small counter argument could be, what if there's no "default" model really as such, but instead you always have to pick exactly one model to install (and potentially extend) in your backend? Maybe that's an idealist POV and the use case of ignoring all default kinds is so esoteric and unusual that it's not worth debating 🤔

Reply:
Thanks for explaining! I agree with you that having a name to define it instead of using "default" makes more sense, as we do for search engines.